include/swift/SIL/GraphFunctionDeviceInfo.h

```diff
@@ -70,13 +97,26 @@ static inline DeviceType getOpDeviceType(llvm::StringRef device) {
   llvm_unreachable("Unknown device type");
 }
 
+static inline DeviceId getOpDeviceId(llvm::StringRef device) {
+  if (device.startswith(TF_CPU_DEVICE_STRING_PREFIX)) {
```

Comment:
Nit: Early-return can make it a little less nested, but either way is fine.

Reply:
Agree. This func missed some code for handling `TF_GPU_DEVICE_STRING_PREFIX`, and I just added it. Thanks!